python/lsst/phosim/utils/phosim_repackager.py

```diff
@@ -44,14 +46,23 @@ class PhoSimRepackager:
     MEFs with one HDU per amp, using LSE-400 for header conformity.
     """
 
-    TELCODE = 'MC'  # Telescope Code: MC is Main Camera
-    CONTRLLR = 'H'  # Controller: H for pHosim
+    CONTRLLR = "H"  # Controller: H for pHosim
 
-    def __init__(self):
+    def __init__(self, prefix="lsst"):
```

Comment:
Please add the doc string of `prefix`. In addition, how about use `instName` or `instrumentNam` or `cameraName` or something more clear than `prefix`?

Reply:
I was following the already-provided argstring structure of `bin.src/phosim_repackager.py` - would you rather see  that changed  from  `--prefix` to `--inst` like in https://github.com/lsst-ts/ts_phosim/blob/5eb96fdfb29ccef77e00e1b47adcf4e1ba081d0f/python/lsst/ts/phosim/CloseLoopTask.py#L983 ? Then I can pass that to `phosim/utils/phosim_repackager.py`

Reply:
I would prefer to use the `--inst` because it is more clear to me. Or maybe you could add the doc string for `prefix` if you want to keep the original one. The bottom line is to make sure this is well-documented in the function's doc string. Thanks!

Reply:
Changed `--prefix` to `--inst` throughout (in `bin.src/phosim_repackager.py` and `phosim/utils/phosim_repackager.py`), and expanded documentation